### DISCUSSION TITLE
Pi-Hole for Windows Discussion

### DISCUSSION THREAD

**Discussion by DesktopECHO** [DesktopECHO/Pi-Hole-for-WSL1#69] [Announcements]
Let me know if it works for you.  I don't collect telemetry so I never can tell!

**Comment by sparkcml** [comment]
Just tested on Win 11.  Installed fine.  Was looking for a way to block ad content on YouTube.  Even with updated lists, YT seems to get it's ad content in.  In all, it says it's blocking lots of ads, but YT must be putting ad content on the same servers as their stream content, so DNS blocking likely won't work.  It would be great to provide some sort of Ad Block Plus integration, so you press a key during an ad, ABP would send a signal to Pi-hole and start blocking that DNS entry.  Just a thought.

**Comment by DesktopECHO** [comment]
Have a look at [this thread](https://discourse.pi-hole.net/t/youtube-ads-getting-through-pihole-any-advances-in-100-blocking-without-also-blocking-youtube-videos/60951#:~:text=Pi%2Dhole%20cannot%20reliably%20block,a%20subscription%20to%20YT%20Premium.).  
Unfortunately Pi-hole cannot reliably block YouTube ads.

(Thanks for letting me know it works otherwise!)

**Comment by sparkcml** [comment]
yeah...good stuff here.  Thanks for all your efforts.  I'm glad to see someone using the Windows/Linux interface process with success.
There used to be a proxy server that could serve as a man-in-the-middle (I used SQUID 30 years ago), and if it was layered over Pi-hole, and used in Transparent mode, it could catch this stuff.  I'm getting   way out of my depth today.  I'm just getting pissed at YT for stopping ad blockers.  It's not like they're not making millions every day.  Greedy SOB's.  As time goes by, I'll just stop using that platform.  I hate ads.
Anyway - Thanks again for your time and effort.

**Reply by DesktopECHO** [reply]
Thanks, I think your best chance would be Firefox + uBlock Origin.
